radicle-node/src/main.rs

```diff
@@ -48,8 +48,17 @@ fn main() -> anyhow::Result<()> {
 
     let options = Options::from_env()?;
     let profile = radicle::Profile::load().context("Failed to load node profile")?;
-    let socket = profile.socket();
-    let client = client::Client::<Reactor>::new(profile).context("Failed to initialize client")?;
+    let node = profile.node();
+    let client = client::Client::<Reactor>::new().context("Failed to initialize client")?;
+    let signer = match profile.signer() {
+        Ok(signer) => signer.boxed(),
+        Err(err) => {
+            let passphrase = env::var("RAD_PASSPHRASE")
+                .context("Either ssh-agent must be initialized, or `RAD_PASSPHRASE` must be set")
+                .context(err)?;
```

Comment:
What about making RAD_PASSPHRASE easier to discover? one could define the env. variable key as a constant with a comment or define or retrieve the env. var at the beginning of the function.

Reply:
Agreed yeah, ideally we also add it to the help output for the command.

Reply:
[092ccfd](https://github.com/radicle-dev/heartwood/pull/74/commits/092ccfdcbc895a73fd5c0efce378fbd5165b4386)